homeassistant/components/rmvtransport/__init__.py

```diff
@@ -1 +1,290 @@
 """The rmvtransport component."""
+import logging
+
+import voluptuous as vol
+
+from homeassistant.components.sensor import PLATFORM_SCHEMA
+from homeassistant.config_entries import SOURCE_IMPORT
+from homeassistant.const import (
+    CONF_NAME,
+    ATTR_ATTRIBUTION,
+    CONF_SCAN_INTERVAL,
+    CONF_SHOW_ON_MAP,
+)
+from homeassistant.exceptions import ConfigEntryNotReady
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
+import homeassistant.helpers.config_validation as cv
+from homeassistant.helpers.dispatcher import async_dispatcher_send
+from homeassistant.helpers.event import async_track_time_interval
+from homeassistant.util import Throttle
+
+# from .config_flow import configured_sensors, duplicate_stations
+from .const import (
+    DEFAULT_SCAN_INTERVAL,
+    DOMAIN,
+    DEFAULT_TIMEOUT,
+    DEFAULT_MAX_JOURNEYS,
+    DEFAULT_TIME_OFFSET,
+    CONF_STATION,
+    CONF_DESTINATIONS,
+    CONF_DIRECTION,
+    CONF_LINES,
+    CONF_PRODUCTS,
+    CONF_TIME_OFFSET,
+    CONF_MAX_JOURNEYS,
+    CONF_TIMEOUT,
+    VALID_PRODUCTS,
+)
+
+_LOGGER = logging.getLogger(__name__)
+
+DATA_RMVTRANSPORT = "rmvtransport"
+DATA_RMVTRANSPORT_CLIENT = "data_rmvtransport_client"
+DATA_RMVTRANSPORT_LISTENER = "data_rmvtransport_listener"
+
+TOPIC_UPDATE = "{0}_data_update".format(DOMAIN)
+
+CONF_NEXT_DEPARTURE = "next_departure"
+
+DEFAULT_NAME = "RMV Journey"
+
+SCAN_INTERVAL = DEFAULT_SCAN_INTERVAL
+
+ICONS = {
+    "U-Bahn": "mdi:subway",
+    "Tram": "mdi:tram",
+    "Bus": "mdi:bus",
+    "S": "mdi:train",
+    "RB": "mdi:train",
+    "RE": "mdi:train",
+    "EC": "mdi:train",
+    "IC": "mdi:train",
+    "ICE": "mdi:train",
+    "SEV": "mdi:checkbox-blank-circle-outline",
+    None: "mdi:clock",
+}
+
+ATTRIBUTION = "Data provided by opendata.rmv.de"
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend(
```

Comment:
Remove the platform schema. The integration should only use the `CONFIG_SCHEMA`.